doc/library/config.txt

```diff
@@ -370,6 +370,33 @@ import theano and print the config variable, as in:
     `amdlibm <http://developer.amd.com/cpu/libraries/libm/>`__
     library, which is faster than the standard libm.
 
+.. attribute:: lib.cnmem
+
+    Float value: >= 0
+
+    Do we enable `CNMeM <https://github.com/NVIDIA/cnmem>`_ or not (a
+    faster CUDA memory allocator). In Theano dev version until 0.7.1
+    is released.
+
+    That library is included in Theano, you do not need to install it.
+
+    The value represent the start size (in MB or % of
```

Comment:
The value represent -> The value represents

Also is that only the initial size and it could grow bigger (as implied by the current text) or is that in fact the maximum size (which is allocated right away).